lib/src/mocks/mock_transports.dart

```diff
@@ -49,9 +49,9 @@ class MockTransports {
   /// a mock expectation or handler is not set up to handle it. This enables
   /// selective mocking - certain requests or WebSockets can be mocked while
   /// the rest will be handled by a real transport platform.
-  static void install({bool fallThrough: true}) {
+  static void install({bool fallThrough: false}) {
```

Comment:
I realized that fallThrough probably shouldn't be enabled by default since the transport mocks will usually be used in unit testing, not app development. Backwards compat is still retained since `configureWTransportForTest()` set `fallThrough` to false anyway